PYME/recipes/processing.py

```diff
@@ -2848,3 +2848,36 @@ def run(self, input_name):
         series_epers.mdh['Camera.ADOffset'] = 0
 
         return series_epers
+    
+
+@register_module('Rotate180')
+class Rotate180(Filter):
+    """
+    Rotate the image 180 degrees
```

Comment:
```suggestion
    Rotate the image 180 degrees in x-y 
```

Just pedantry - as recipe modules are often used with 3D data it's otherwise slightly ambiguous